src/WinRT.Runtime2/InteropServices/ExceptionErrorInfo.cs

```diff
@@ -0,0 +1,150 @@
+{
+    public static unsafe void GetErrorDetails(
+        void* thisPtr,
+        out string description,
+        out int error,
+        out string restrictedDescription,
+        out string capabilitySid)
+    {
+        void* _description = null;
+        void* _restrictedDescription = null;
+        void* _capabilitySid = null;
+
+        try
+        {
+            fixed (int* pError = &error)
+            {
+                // GetErrorDetails
+                Marshal.ThrowExceptionForHR(((delegate* unmanaged[Stdcall]<void*, void**, int*, void**, void**, int>)(*(void***)thisPtr)[3])(
+                    thisPtr,
+                    &_description,
+                    pError,
+                    &_restrictedDescription,
+                    &_capabilitySid));
+            }
+
+            description = _description != null ? Marshal.PtrToStringBSTR((nint)_description) : string.Empty;
+            restrictedDescription = _restrictedDescription != null ? Marshal.PtrToStringBSTR((nint)_restrictedDescription) : string.Empty;
+            capabilitySid = _capabilitySid != null ? Marshal.PtrToStringBSTR((nint)_capabilitySid) : string.Empty;
+        }
+        finally
+        {
+            Marshal.FreeBSTR((nint)_description);
+            Marshal.FreeBSTR((nint)_restrictedDescription);
+            Marshal.FreeBSTR((nint)_capabilitySid);
+        }
+    }
+
+    public static unsafe void GetErrorDetails(
+        void* thisPtr,
+        out int error)
+    {
+        void* _description = null;
+        void* _restrictedDescription = null;
+        void* _capabilitySid = null;
+
+        try
+        {
+            fixed (int* pError = &error)
+            {
+                // GetErrorDetails
+                Marshal.ThrowExceptionForHR(((delegate* unmanaged[Stdcall]<void*, void**, int*, void**, void**, int>)(*(void***)thisPtr)[3])(
+                    thisPtr,
+                    &_description,
+                    pError,
+                    &_restrictedDescription,
+                    &_capabilitySid));
+            }
+        }
+        finally
+        {
+            Marshal.FreeBSTR((nint)_description);
+            Marshal.FreeBSTR((nint)_restrictedDescription);
+            Marshal.FreeBSTR((nint)_capabilitySid);
+        }
+    }
+
+    public static unsafe string GetReference(void* thisPtr)
+    {
+        void* __retval = default;
+
+        try
+        {
+            // GetReference
+            Marshal.ThrowExceptionForHR(((delegate* unmanaged[Stdcall]<void*, void**, int>)(*(void***)thisPtr)[4])(
+                thisPtr,
+                &__retval));
+            return __retval != null ? Marshal.PtrToStringBSTR((nint)__retval) : string.Empty;
+        }
+        finally
+        {
+            Marshal.FreeBSTR((nint)__retval);
+        }
+    }
+}
+
+internal static class ILanguageExceptionErrorInfoMethods
+{
+    public static unsafe void* GetLanguageException(void* thisPtr)
```

Comment:
Direct stubs like this one that directly just invoke a method with no additional marshalling on an untyped pointer can just be added to the vtable type. E.g. delete this type and move the logic for this method to `ILanguageExceptionErrorInfoVftbl.GetLanguageExceptionUnsafe`. The structure can match what e.g. `IInspectableVftbl.GetRuntimeClassNameUnsafe`.